common/common.go

```diff
@@ -261,9 +261,9 @@ func (ud *UserData) Set(id string, in interface{}) error {
 
 // Update updates existing UserData with new UserData.
 // If a key as an empty string, the key will be deleted
-func (ud *UserData) Update(new *UserData) {
+func (ud *UserData) Update(new *UserData) *UserData {
```

Comment:
Could we add godoc as to why this returns a pointer? Isn't it self-mutating as well?

Reply:
It returns a pointer so it can be chained, i.E. `ud.Update(otherUD).ToMap()`